service/src/main/kotlin/fi/espoo/evaka/specialdiet/SpecialDietController.kt

```diff
@@ -0,0 +1,91 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.specialdiet
+
+import fi.espoo.evaka.Audit
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.EvakaClock
+import fi.espoo.evaka.shared.security.AccessControl
+import fi.espoo.evaka.shared.security.Action
+import org.springframework.web.bind.annotation.*
+
+data class JamixSpecialDiet(val modelId: Int, val fields: JamixSpecialDietFields)
+
+data class JamixSpecialDietFields(val dietName: String, val dietAbbreviation: String)
+
+fun cleanupJamixDietList(specialDietList: List<JamixSpecialDiet>): List<SpecialDiet> {
```

Comment:
consider adding a simple unit test for this to verify different cases

Reply:
I was planning to add tests for this once I move this code to it's final location. e.g. when we are doing the Jamix GET /diets API call directly from eVaka. For now this is just a quickly made tool which implementation details might change before production use.